src/status_im2/contexts/shell/jump_to/gesture.cljs

```diff
@@ -39,5 +29,5 @@
            :screen-width           width
            :left-velocity          constants/gesture-fling-left-velocity
            :right-velocity         constants/gesture-fling-right-velocity
-           :screen-closed-callback (screen-closed-callback screen-id)})))))
+           :screen-closed-callback screen-closed-callback})))))
```

Comment:
nit: consider changing name to `on-screen-closed`

Reply:
done, thank you